index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Intuit Accounts - Sign Up</title>
+    <script src="https://unpkg.com/ionicons@5.0.0/dist/ionicons.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="assets/favicon.png"/>
+</head>
+<body>
+    <header>
+        <div class="d-flex signin-wrapper">
+            <div class="signin-text-wrapper">
+                <div>Already have an Intuit account?</div>
+                <div><a href="#">I forgot my User ID or Password</a></div>
+            </div>
+            <button type="button" class="signin-button"><ion-icon class="icon-svg" name="lock-closed-outline"></ion-icon>Sign In</button>
+        </div>
+        <div class="d-flex logos-wrapper">
+            
+            <img class="intuit-logo" src="assets/logo-intuit.png" alt="intuit logo">
+            <div>
+                <img class="logos" src="assets/header-icon.png" alt="header logos"/>
+            </div>

```

Comment:
- [ ]  It should show the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [ ]  It should show the Quickbooks logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [ ]  It should show the Turbotax logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))